### ISSUE TITLE
User name

### ISSUE THREAD

**Issue by gitbizzy** [LOS-zz/PUD#1]
HI i noticed that you dont use github.  I wanted to know if you were willing to let me have your username?  please respond. Thank you

**Comment by eadric** [comment]
lol, didnt know i had this account.  sup mang? I'm actually writing code at work these days, left IT and moved to our engineering group.  Writing python to glue shit together right now

**Comment by gitbizzy** [comment]
lol word.  I thought maybe you didn't even have your email linked to it since no action in like 6 years.  But yea, we use github at my job n my name is Carlos so I really want that screenname can you do it for me?  If you can email me solrac.g83@gmail.com

**Comment by LOS-zz** [comment]
Lolz

**Comment by gitbizzy** [comment]
o man i didn't realize i was messaging the wrong guy.  but wsup los, just switch the email on the account to solrac.g83@gmail.com  and ill reset the password. Ill even give u $10 via paypal for the trouble lol

**Comment by gitbizzy** [comment]
so wsup.  you gonna let me have it or no?  the suspense is kiling me lol.  but let me know either way, i would really appreciate it.  n like i said your not using it anyway.  plz respond thnx

**Comment by LOS-zz** [comment]
what exactly are you looking for? the LOS username?

LoS has been active since 1998.

**Comment by gitbizzy** [comment]
Yeah the github username, so i can login as Los and my github would be github.com/los.  Since you dont seem to use git, I figured you would probably be willing to just hand it down.  i mean, you said yourself you didnt even know you had this account.  if that was u. I don't know if maybe you have a reason to hold on to it?  Github has only been active since 2008 and it says you signed up in 2009.  1998?

**Comment by gitbizzy** [comment]
Ok, for whatever reason, it seems like your not willing to give up the account, or even answer if you will or wont.  I don't want to bother you, n I apologize for blowing up your email, but I figured I'd ask one last time.  Again, since you don't use it and didn't even remember you had it, if you could just switch the email on the github.com/los account to solrac.g83@gmail.com and comment here that its done, I would really appreciate it. Again my bad if I've been bothering you but you never really said yes or no so Ive been trying to get an answer. If you celebrate it, Happy Easter. :rabbit:   If not have a good day.

**Comment by gitbizzy** [comment]
cmon guy.  ur gonna let such a great name waste away to oblivion lol.  Just let me have it. or at least give me one good reason why you dont want to. I don't get it

**Comment by gitbizzy** [comment]
Really?   Ur not gonna give it to me n just let it waste away for no reason? Really? I mean seriously why not? :smiling_imp:

**Comment by Los83** [comment]
I see you still don't use this account 🙃 I really want this account can u please consider my proposal since you don't use it

**Comment by eadric** [comment]
Please name at least five manly meats, in order of manliness and your
request will be considered.

**Comment by gitbizzy** [comment]
1. Beef heart - Because it just sounds cool.
2. Bear - Only if you killed it yourself.
3. Opossum - Because anyone who eats that is twice the man I am.
4. Calf liver - Because if you eat babies you're a badass.
5. Alligator
